src/email-templates/new_company_application_company.handlebars

```diff
@@ -1,8 +1,11 @@
-<h1>We have successfully received your application!</h1>
-<p>We will now review your application, and in case you're approved, you will receive another email with further instructions in order to complete your registration.</p>    
-<p>Your Application ID is {{applicationId}} and you registered {{companyName}}</p>
+<h1>Your application has been validated!</h1>
+<p>We will now review your application to determine the trustworthiness of your company.
+    In the meantime, you can complete your registration by logging into your account and begin creating offers.
+    Please note that your offers will remain hidden from the public until your company is approved.</p>
+<p>Your Application ID is {{applicationId}} and you registered {{companyName}}.</p>
```

Comment:
Don't say "trustworhiness", I think you can just say that we are verifying or reviewing the application.

Reply:
@FranciscoCardoso913 any comments on this?